app/Providers/AppServiceProvider.php

```diff
@@ -23,6 +24,10 @@ public function register()
      */
     public function boot()
     {
-        //
+        View::composer('*', function ($view) {
```

Comment:
I'm not a fan of the view composer because it is not obvious where that comes from. What about we put that method to the layout component?

Reply:
That's right. I decided to add a php component class and put the logic into it. Does this make sense for you?

Reply:
Much better yes 👍